### PR TITLE
Fixes For 'IOC Alert' XSIAM Playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10789

## Description
There are two unnecessary manual tasks in the playbook: 
Task number 43 - 'Should continue with the investigation?' - Asks the user whether the alert investigation should continue after the investigation or remediation stages.
Task number 44 - 'Continue with the alert investigation' - Asks the user to continue with the alert investigation.

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 1]
```diff
@@ -1136,8 +1072,8 @@ tasks:
     task:
       id: 37a6525e-26a7-4c1f-8531-17fe493b2642
       version: -1
-      name: Set Incident Severity to High
-      description: commands.local.cmd.set.parent.incident.field
+      name: Set Alert Severity to High
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1075]: Please reverse this change.

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml [hunk 2]
```diff
@@ -1136,8 +1072,8 @@ tasks:
...
+      description: commands.local.cmd.set.parent.alert.field
```
Comment (OmriItzhak) [inline Packs/Core/Playbooks/playbook-IOC_Alert.yml:1076]: Please reverse this change.

File: Packs/Core/doc_files/IOC_Alert.png
Comment (OmriItzhak) [inline Packs/Core/doc_files/IOC_Alert.png:1]: Please change the theme in the playbook picture to white.

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml (hunk 1 above)
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-IOC_Alert.yml]: done

File: Packs/Core/doc_files/IOC_Alert.png
Comment (TalNos) [reply Packs/Core/doc_files/IOC_Alert.png:1]: done

File: Packs/Core/Playbooks/playbook-IOC_Alert.yml (hunk 2 above)
Comment (TalNos) [reply Packs/Core/Playbooks/playbook-IOC_Alert.yml]: done

**Review by OmriItzhak — APPROVED** [review]

**Pull request merged**